wdn/templates_4.1/scripts/main-wdn-plugins.js

```diff
@@ -1,8 +1,8 @@
 define([
 	'analytics',
+	'socialmediashare',
 	'navigation',
 	'search',
-	'socialmediashare',
```

Comment:
I thought order does matter because of the for loop with `pluginObj.initialize();`?